src/Components/CreateImageWizard/formComponents/FileSystemConfiguration.js

```diff
@@ -52,6 +54,45 @@ const FileSystemConfiguration = ({ ...props }) => {
   const bodyref = useRef();
   const [rows, setRows] = useState([initialRow]);
 
+  const oscapPolicy = getState()?.values?.['oscap-policy'];
+  const hasNoOscapPolicy = !oscapPolicy;
+  const hasCustomizations = !(
+    getState()?.values?.['file-system-configuration'] === undefined ||
+    getState().values['file-system-configuration'].length === 1
+  );
+
+  const { data: customizations, isSuccess } = useGetOscapCustomizationsQuery(
+    {
+      distribution: getState()?.values?.['release'],
+      profile: oscapPolicy,
+    },
+    {
+      // Don't override the user's data if they made customizations
+      skip: hasNoOscapPolicy || hasCustomizations,
+    }
+  );
+
+  useEffect(() => {
+    if (hasCustomizations || rows.length > 1) {
+      return;
+    }
+    if (customizations && customizations.filesystem && isSuccess) {
```

Comment:
We get duplicated `/` mount point when it's a part of fs customizations:

"xccdf_org.ssgproject.content_profile_cis_server_l1"
![Screenshot from 2023-10-06 10-50-25](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/3afe6555-abbc-45dd-9b1a-e8a5a2bbdd05)

The question is which one should we be keeping? Also should there be some kind of size check on change when a minimal size is set?

Reply:
This is now patched up. Thanks for finding this!